CDP4JsonFileDal/JsonFileDal.cs

```diff
@@ -176,7 +176,7 @@ public override Task<IEnumerable<Thing>> Write(IEnumerable<OperationContainer> o
             var iterations = new HashSet<CDP4Common.EngineeringModelData.Iteration>();
             var siteReferenceDataLibraries = new HashSet<CDP4Common.SiteDirectoryData.SiteReferenceDataLibrary>();
             var modelReferenceDataLibraries = new HashSet<CDP4Common.SiteDirectoryData.ModelReferenceDataLibrary>();
-            var domainsOfExpertises = new HashSet<CDP4Common.SiteDirectoryData.DomainOfExpertise>();
+            var domainsOfExpertise = new HashSet<CDP4Common.SiteDirectoryData.DomainOfExpertise>();

```

Comment:
The plural version of "domain of expertise" is indeed "domain**s** of expertise".
From a readability point of view I find domainOfExpertise**s** a better choice as a variable name. 
I noticed that I had more problems reading the AddDomainsOfExpertise method. I somehow read over the 's' in domains. The difference between singural and plural seems easier when the plural version ends with an 's'.

Reply:
This is not what I meant. Please keep the **s** at the end of the variable name.